jupyterhub/auth.py

```diff
@@ -87,7 +94,28 @@ class LocalAuthenticator(Authenticator):
         should I try to create the system user?
         """
     )
-    
+
+    whitelist_group = Unicode(
+        config=True,
+        help="Automatically whitelist anyone in this group.",
+    )
+
+    def check_whitelist(self, username):
+        return (
+            super().check_whitelist(username) or
```

Comment:
If the group whitelist is a set as well, I don't think there's anything you can do with names that you can't do with groups. Of course, a special-purpose 'jupyterhub' group would also cover it.